code/game/machinery/doors/airlock.dm

```diff
@@ -1109,13 +1109,15 @@ About the new airlock wires panel:
 	if(istype(I, /obj/item/twohanded/fireaxe)) //let's make this more specific //FUCK YOU
 		var/obj/item/twohanded/fireaxe/F = I
 		if(F.wielded)
-			spawn(0)
-				if(density)
-					open(1)
+			if(density)
+				playsound(src, 'sound/machines/airlock_alien_prying.ogg', 100, 1) //is it aliens or just the CE being a dick?
+				prying_so_hard = TRUE
+				if(do_after(user, 75, target = src))
+					prying_so_hard = FALSE
```

Comment:
You need to reset `prying_so_hard = FALSE` even if `do_after` fails. Otherwise, interrupting the prying would make it unpriable...
.. but only by jaws of life, since you're not checking prying_so_hard here at all.

And now that I say that, this is effectively a copy-paste of jaws behavior several lines below. Let's not have copy-paste like that. Either see that the code falls through to the jaws when using fireaxe OR isolate this logic into a dedicated proc, and just call that proc in these two places.


Also add a comment on the variable what it means, because the name is atrocious. "Whether the door is currently being pried by someone" or similar.

Reply:
Made the prying so hard work, kept it different as one uses the two handed.